src/races.cpp

```diff
@@ -1243,7 +1243,42 @@ RaceDef races[NUM_RACES] = {
      100,                       /* AC factor for mobs */
      nullptr,                   /* verb for entering a room */
      nullptr,                   /* verb for walking out of a room */
-     {0, 0}}                   /* function numbers - always set as {0, 0}}, to close struct */
+     {0, 0}},                   /* function numbers - always set as {0, 0}}, to close struct */
+
+    /* ARBOREAN */
+    {"arborean",                /* name as found by triggers */
+     "arborean",                /* all race keywords */
+     "&2Arborean&0",            /* name as displayed at character creation, on who, and score */
+     "&2Arborean&0",            /* name as displayed in show race command and when setting races */
+     "Arborean",                /* name as displayed in medit vsearch stat and enlightenment */
+     false,                      /* playable? */
+     true,                      /* humanoid? */
```

Comment:
Are they humanoids?

Reply:
yes, they're basically Ents.  They also need to be humanoid or else there are a bunch of class skills they can't use...